package.json

```diff
@@ -36,7 +37,7 @@
         "@typescript-eslint/eslint-plugin": "^5.20.0",
         "@typescript-eslint/parser": "^5.20.0",
         "@wharfkit/mock-data": "^1.2.0",
-        "@wharfkit/session": "^1.4.0-rc3",
```

Comment:
I was seeing some type mismatch errors when using the newer session kit version.

Reply:
Yeah I wouldn't use a RC version for an update like this. Those are subject to change and might have bugs.

Reply:
Yeah - it requires the RC version to work. The 1.4.0 release introduces the key lookup mechanism that this plugin requires.

Reply:
What key lookup mechanism? 😅 I think I might have missed that part lol.

Reply:
The web-renderer (1.4.0-rc6) and session (1.4.x) does automatic key lookups now if a wallet provides a public key method:

https://github.com/wharfkit/web-renderer/blob/75b432fc77a57b455ffc72ee5384a988a66b4d6b/src/ui/login/Permission.svelte#L37-L45

That automatically retrieves the key and then does a lookup, displaying the accounts you can login with.

<img width="590" alt="image" src="https://github.com/user-attachments/assets/02a88b59-6047-4307-9e0e-943a5b895ad4">

The metamask plugin implements this method, so it just automatically does lookups.

https://github.com/wharfkit/wallet-plugin-metamask/blob/ae23425daead0ec4654b01ff57ecba3c7f67617c/src/index.ts#L93-L103

This part of the web-renderer might be a place we could inject some sort of plugin to create an account?

Reply:
Oh sweet! And for the account creation, we would need to display a button to trigger that somewhere, right? I was thinking that we could have the session kit, handle displaying that button when a compatible account creation plugin is available.

Reply:
Ideally yeah - if a creation plugin was available, we could show it in a few different spots. 

I don't think the account creation plugin UI was developed like that though - and instead I think it was meant to be manually triggered by the developer 🤔 

So we may need some web-renderer changes.

Reply:
Yeah, right now you need to trigger it with `sessionKit.createAccount()`. And yeah, we probably need to make some changes to the web renderer to make the UX better on that front.